### PR TITLE
feat: integration cards: support list with numeric header

### PR DESCRIPTION
**Pull request by MarcoEidinger**
https://ui5.sap.com/test-resources/sap/ui/integration/demokit/cardExplorer/webapp/index.html#/explore/list/numeric

also fixes numeric header to consider state in addition
to trend to determine color

listnumeric

### CODE REVIEW

**Comment by MarcoEidinger** [comment]
PR is ready for review and merge

Please note why failing checks can be ignored for now:
- code formatting check fails due to incorrect formatted FioriChart sources (not part of this PR). Sheng will fix these in a future PR
- snapshot testing due to
  - new test and missing reference image
  - changed behavior (consider state for KPI color determination and KPI unit of measurement shall be color-coded as well) 

 I'll record new snapshot reference images once this PR is merged (as I can do this via CI capabilities)

File: Sources/FioriIntegrationCards/Components/Common/Info.swift [hunk 1]
```diff
@@ -0,0 +1,37 @@
+import SwiftUI
+
+public final class Info: Decodable, AnyBodyProducing {
+    var value: String?
+    var state: String?
+
+    func body() -> AnyView {
+        AnyView(
+            Text(self.value ?? "")
```
Comment (ShawnMa16) [inline Sources/FioriIntegrationCards/Components/Common/Info.swift:9]: Probably we can use the `SafeText` here I guess?

**Review by ShawnMa16 — CHANGES_REQUESTED** [review]

File: Sources/FioriIntegrationCards/Components/Common/Info.swift (hunk 1 above)
Comment (MarcoEidinger) [reply Sources/FioriIntegrationCards/Components/Common/Info.swift:9]: For some reason, it cannot find `SafeText`. Does it work for you?

Comment (ShawnMa16) [reply Sources/FioriIntegrationCards/Components/Common/Info.swift:9]: I just tried, it won't show up either in my Xcode

**Review by ShawnMa16 — CHANGES_REQUESTED** [review]

File: Sources/FioriIntegrationCards/Components/Common/Info.swift (hunk 1 above)
Comment (MarcoEidinger) [reply Sources/FioriIntegrationCards/Components/Common/Info.swift]: I pushed a commit which allows the use of `SafeText` without an instance of View and then I use it in this example

**Review by ShawnMa16 — APPROVED** [review]

**Pull request merged**